Pod/Classes/URBNExtraTextHighlightedTextStorage.h

```diff
@@ -0,0 +1,15 @@
+//
+//  URBNExtraTextHighlightedTextStorage.h
+//  Pods
+//
+//  Created by Nick DiStefano on 6/19/15.
+//
+//
+
+#import <UIKit/UIKit.h>
+
+@interface URBNExtraTextHighlightedTextStorage : NSTextStorage
+
+- (instancetype)initWithErrorTextColor:(UIColor *)errorTextColor maxLength:(NSInteger)maxLength;
```

Comment:
So in that case the above method should maybe be renamed or tossed all together.